consensus/ibft/hooks.go

```diff
@@ -82,6 +86,9 @@ const (
 	// CalculateProposerHook defines what is the next proposer
 	// based on the previous
 	CalculateProposerHook = "CalculateProposerHook"
+
+	// InsertTransactionsHook defines the process to insert additional transactions
+	InsertTransactionsHook = "InsertTransactionsHook"
```

Comment:
You can add a comment marking it is only used for the Bridge mechanism:
`// BRIDGE //`

Reply:
@zivkovicmilos To be honest, I don't like IBFT knows what hooks are for each mechanism because mechanism is extension and IBFT must not know what extension is.